cypress/e2e/fill-form.cy.ts

```diff
@@ -0,0 +1,62 @@
+import {StudentFormPage} from "../page";
+
+
+describe("This test will fill a form", () => {
+  // Arrange
+  const personalInformation = {
+    name: "Holmes",
+    lastName: "Salazar",
+    email: "test@email.com",
+    gender: "Male",
+    dateOfBirth: "27 Jul 2016",
+    mobileNumber: "3656589156",
+    hobbies: ["Music", "Reading"],
+    currentAddress: "Av siempreViva # 123",
+  };
+
+  const month = [
+    "January",
+    "February",
+    "March",
+    "April",
+    "May",
+    "June",
+    "July",
+    "August",
+    "September",
+    "October",
+    "November",
+    "December",
+  ];
```

Comment:
you don't need this, if you use the method `toLocaleString("default", { month: "long", });` of the date object you can get the right name of month needed in validation